extensions/ql-vscode/src/variant-analysis/ghec-dr.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  VARIANT_ANALYSIS_ENABLE_GHEC_DR,
+  hasEnterpriseUri,
+  hasGhecDrUri,
+} from "../config";
+
+/**
+ * Determines whether MRVA should be enabled or not for the current GitHub host.
+ * This is based on the `github-enterprise.uri` setting.
+ */
+export function isVariantAnalysisEnabledForGitHubHost(): boolean {
```

Comment:
Would it make sense to make this part of the `VariantAnalysisConfig`?

Reply:
I think right now it wouldn't make any difference because that config is only used to tell `DbTreeDataProvider` when to update, and I believe that the GHEC-DR setting wouldn't affect that view.
Since `VariantAnalysisConfig` only contains a couple of fields and by no means all of the variant analysis config fields, I'm tempted to also not add this one right now. 🤷🏼